app.rb

```diff
@@ -1,9 +1,20 @@
 require 'sinatra/base'
+require 'pg'
 
 class Chitter < Sinatra::Base
   get '/test' do
     'Test page'
   end
 
+  get '/' do
+    'Welcome to Chitter'
+  end
+
+  get '/peeps' do
+    Chitter.add
```

Comment:
This route should return a template that shows the user all of the peeps in the database. That means you'll need to use Peep.all at some point.